explorer/app/entity/list/ListContainer.tsx

```diff
@@ -29,6 +37,13 @@ export const ListContainer = (props: ListModel) => {
     <TableCreator<DetailResponseType>
```

Comment:
Should this be `<ListResponseType>` here?

Reply:
this is actually correct. The Table creator uses a generic type to assume that it will receive an array of that type.  ListResponseTypes are just one object with the `hits` field

Reply:
Got it, thanks Bruce!